products/compute/api.yaml

```diff
@@ -11905,6 +11905,9 @@ objects:
           contained in this subnetwork. The primary IP of such VM must belong
           to the primary ipCidrRange of the subnetwork. The alias IPs may belong
           to either primary or secondary ranges.
+          **Note**: This field will default to the API if empty. In order to explicitly
+          remove any secondary IP ranges, set this to an empty array. 
```

Comment:
You should be adding this to the [terraform.yaml](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/compute/terraform.yaml#L1845) as this information isn't relevant for non-terraform providers.

Reply:
I also have a slight preference that you say the empty list syntax only works because of the `Attr-as-Block` designation. All other lists will throw a syntax error if used this way.